kythe/docs/schema/indexing-protobuf.txt

```diff
@@ -31,12 +31,235 @@ implementation, as this is often difficult (or uninteresting) to read. This
 document describes how to encode information about generated code to permit
 cross-language links.
 
+To make the discussion easier to understand let's pretend we are working with
+two languages: AliceLang and BobLang. AliceLang has `.alice` file and BobLang
```

Comment:
SourceLang TargetLang?

Alice and Bob are commonly used examples, but it's not necessarily obvious which is generated from the other.  It might be beneficial to use more descriptive names?

Reply:
Renamed. Yes SourceLang and TargetLang are easier to understand.